test/test_qt_utils.py

```diff
@@ -0,0 +1,229 @@
+"""
+Test Module for qt_utils
+"""
+import numpy as np
+import pytest
+from c3.utils.qt_utils import (
+    pauli_basis,
+    basis,
+    xy_basis,
+    get_basis_matrices,
+    rotation,
+    np_kron_n,
+    hilbert_space_kron,
+    kron_ids,
+    projector,
+    pad_matrix,
+    perfect_parametric_gate,
+    T1_sequence,
+    ramsey_sequence,
+    ramsey_echo_sequence,
+)
+from numpy.testing import assert_array_almost_equal as almost_equal
+from c3.libraries.constants import GATES
+
+
+@pytest.fixture()
```

Comment:
Great. Thanks for providing a fixture for this test set. All fixtures should be located in the `test/conftest.py` file. Can you update this accordingly?

Reply:
Yes, I didn't see that file before. Moved it there now